Last-Update: 2012-08-24
Author: Dmitry Smirnov <onlyjob@member.fsf.org>
Forwarded: not-needed
Description: backported patch (r31748)
 Fix crash with copy & paste.
 Don't provide a NULL pointer as callback function.
 It's a regression from r30427.
 This closes bugzilla bug 13329.

--- a/src/af/xap/gtk/xap_UnixClipboard.cpp
+++ b/src/af/xap/gtk/xap_UnixClipboard.cpp
@@ -309,8 +309,6 @@
 	GtkClipboard * clipboard = gtkClipboardForTarget (tFrom);
 #if DEBUG
 	gtk_clipboard_request_targets(clipboard,( GtkClipboardTargetsReceivedFunc) allTargets, this);
-#else
-	gtk_clipboard_request_targets(clipboard, NULL, NULL);
 #endif
 	UT_GenericVector<GdkAtom> atoms ;
 	for(int atomCounter = 0; formatList[atomCounter]; atomCounter++)
